src/batch-client/types.ts

```diff
@@ -21,6 +21,12 @@ export interface AutoCompleteOptions {
 	needExp?: boolean;
 	type?: GalSearchType;
 }
+export interface AutoCompleteGALOptions {
+	limit: number;
```

Comment:
name is only required input, all others should be optional

Reply:
Changed and pushed